vm/systemSmartContracts/delegation_test.go

```diff
@@ -367,7 +366,7 @@ func TestDelegationSystemSC_ExecuteInitShouldWork(t *testing.T) {
 
 	ok, delegator, err := d.getOrCreateDelegatorData(ownerAddr)
 	assert.Nil(t, err)
-	assert.True(t, ok)
+	assert.False(t, ok)
```

Comment:
Please resolve the weak warning at L835.
the function `getSuccessAndUnSuccessKeys` needs more testing

Reply:
Done.